README.md

```diff
@@ -92,21 +92,39 @@ url_path:
 
 ## Run the application locally
 
-Ensure you have the required tools to create Python virtual environments. You might need to install a specific package depending on your distribution, e.g. on Debian `python3-venv`:
+Ensure you have the required tools to create Python virtual environments.
+You can either install the dependencies globally, or use [devbox](https://www.jetpack.io/devbox).
+
+### Install dependencies globally
```

Comment:
"globally" usually means "for all users on the system", so I'd change that to "Install dependencies in a virtualenv"